src/main/kotlin/com/sourcegraph/cody/config/SettingsMigration.kt

```diff
@@ -29,6 +30,7 @@ class SettingsMigration : Activity {
       migrateProjectSettings(project)
       migrateAccounts(project)
     }
+    RunOnceUtil.runOnceForProject(project, "CodyHistoryLlmMigration") { migrateLlms(project) }
```

Comment:
Why this migration is in-between other migrations? I think logically this migration should be performed last.

Reply:
It's the last of other `runOnceForProject`s. The later entries are `runOnceForApp`.

In general, I agree - we should append the migration e.g.:
1. A->B
2. B->C
The other way it would not work (assuming that dependency exists).

However `runOnceForProject` is not the same as `runOnceForApp` - those are different properties handled by different xml files. IMHO we are OK in this case.

Reply:
Sure! Thanks for explanation.